handbook/people.md

```diff
@@ -564,21 +564,21 @@ A tweet storm is a series of Twitter posts usually made as replies in a single t
 TODO
 
 ### Responsibilities
- TODO: this is a chart outlining responsibilities based on meeting types
+ TODO: this is a chart outlining responsibilities based on meeting types.
 
 ### Email best practices
-TODO: In order to ensure continuity across CEO shadow participants. Always, cc TDOD on emails as part of the program. This ensures that even after you've left the program the response and follow-up can be tracked
+TODO: In order to keep continuity across CEO shadow participants. Always, cc TDOD on emails as part of the program. This makes sure that even after you've left the program, the response and follow-up can be tracked.
```

Comment:
What does TDOD mean?

Reply:
This is an email group that needs to be created.

Reply:
I was wondering if you meant "TODO."

Reply:
Yeah, definitely a typo. lol